contrib/bash_compl/_rgbfix.bash

```diff
@@ -21,6 +21,7 @@ _rgbfix_completions() {
 		[l]="old-licensee:unk"
 		[m]="mbc-type:mbc"
 		[n]="rom-version:unk"
+		[o]="output:normal"
```

Comment:
This should be
```suggestion
		[o]="output:glob-*.gb?"
```
Since the option takes an argument.

Reply:
Done.